public/app/core/services/echo/backends/PerformanceBackend.ts

```diff
@@ -22,8 +34,12 @@ export class PerformanceBackend implements EchoBackend<PerformanceEvent, Perform
 
   constructor(public options: PerformanceBackendOptions) {}
 
-  addEvent = (e: EchoEvent) => {
-    this.buffer.push(e.payload);
+  private shouldBeSentToBackend = (e: PerformanceEvent) => !isLiveEvent(e);
```

Comment:
Does it make sense to reuse this class/structure for live testing?  This is really aimed at collecting stats from end user usage (in cloud etc)

If we are never going to send the live events to the performace backend... then maybe better if we use a different capture method?

Reply:
refactored